### ISSUE TITLE
v3.0.4 can't get en_core_web_sm

### ISSUE THREAD

**Issue by ghk829** [explosion/spaCy#7379]
## How to reproduce the behaviour

``` sh
python -m spacy download en_core_web_sm
```
then I got a message
```
No compatible package v3.0.4 of spacy
```

## Your Environment

* Operating System: Linux
* Python Version Used: 3.7
* spaCy Version Used: 3.0.4
* Environment Information: root

**Comment by polm** [comment]
Sorry about that, we *just* released 3.0.4. Please wait a little while for us to update the compatibility list.

**Comment by polm** [comment]
OK, should be fixed now. Thanks for reporting!

Note that even after the compatibility list is updated, sometimes git caches the URL for a while, so that can result in errors like this. With just-released versions sometimes you may need to wait a little while and try again unfortunately.
